FAQ.md

```diff
@@ -0,0 +1,23 @@
+**FAQ**
+
+**What am I looking at?**
+
+You're looking at an atmospheric temperature profile. The air temperature is calculated at three points: at the surface, at a first middle level, and at a second level at the top of the atmosphere.
+
+**Why is it zero all the way up?**
+
+We haven't given the model any information yet. For instance, the starting value for the solar constant, which is how much energy (light) from the sun is reaching Earth, is set at 10 Watts per square meter (W/m^(-2)). With such a low energy input, the Earth would be very cold. Change the value for the solar constant to about 1370 W/m^(-2) – that's about how much it is in real life.
```

Comment:
Agreed, though I think it's simply because the button 'Calculate Model' hasn't been pressed yet. But yea, it would be cold with such a low solar constant (like off the plot cold!).